samples/drivers/mbox/sample.yaml

```
@@ -23,6 +23,47 @@ tests:
         - "Pong \\(on channel 0\\)"
         - "Ping \\(on channel 1\\)"
         - "Pong \\(on channel 1\\)"
+
+  sample.drivers.mbox.nrf54h20_vevif:
+    platform_allow:
+      - nrf54h20pdk_nrf54h20_cpuapp
+    integration_platforms:
+      - nrf54h20pdk_nrf54h20_cpuapp
+    extra_args:
+      mbox_SNIPPET=nordic-ppr
+      mbox_OVERLAY_CONFIG=boards/nrf54h20pdk_nrf54h20_cpuapp_vevif.conf
```

`OVERLAY_CONFIG` is deprecated.
```suggestion
      mbox_EXTRA_CONF_FILE=boards/nrf54h20pdk_nrf54h20_cpuapp_vevif.conf
```

---

done